engine/apps/api/views/auth.py

```diff
@@ -25,6 +26,8 @@ def overridden_login_slack_auth(request, backend):
     # We can't just redirect frontend here because we need to make a API call and pass tokens to this view from JS.
     # So frontend can't follow our redirect.
     # So wrapping and returning URL to redirect as a string.
+    if settings.SLACK_INTEGRATION_MAINTENANCE:
```

Comment:
May be rename it to
```suggestion
    if settings.MAINTENANCE_FOR_SLACK_MESSAGING_BACKEND_ENABLED:
```

Reply:
I started name from SLACK since all slack related envs are named with that convention. It's called INTEGRATION, because we have FEATURE_SLACK_INTEGRATION_ENABLED, so I wanted to be consistent here too. I'm agree, that to be consistent it's better to add ENABLED to the end, as we do in other places.

Reply:
I had two points
* booleans end with _ENABLED
* MAINTENANCE be confused with maintenance mode in OnCall